docs/src/developer_documentation/custom_themes.md

```diff
@@ -273,91 +296,91 @@ Here's what that looks like for the frontmatter entries:
 ```typ
 // foo/components/toc.typ
 
-#import "/utils.typ"
-
-#let toc() = utils.print-toc((_, body, appendix) => {
-  // replace frontmatter with _
-  // to indicate we aren't using it
+// TODO: import utils
 
+#let toc = utils.make-toc((_, body, appendix) => {
```

Comment:
I think you should still include the `// replace frontmatter with _ to indicate we aren't using it` comment